rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -29,14 +35,26 @@ def __init__(self, play_selector: situation.IPlaySelector, world_state: Optional
         self.world_state_sub = self.create_subscription(msg.WorldState, '/vision_filter/world_state', self.create_partial_world_state, 10)
         self.field_dimenstions = self.create_subscription(msg.FieldDimensions, '/config/field_dimensions', self.create_field, 10)
         self.game_info = self.create_subscription(msg.GameState, '/referee/game_state', self.create_game_info, 10)
+
+        self.robot_state_subs = [None] * NUM_ROBOTS
+        self.robot_intent_pubs = [None] * NUM_ROBOTS
+
+        self.override_actions = [None] * NUM_ROBOTS
+
+
+        for i in range(NUM_ROBOTS):
+            self.robot_state_subs[i] = self.create_subscription(msg.RobotStatus, '/radio/robot_status/robot_'+str(i), self.create_partial_robots, 10)
+ 
         for i in range(NUM_ROBOTS):
-            self.game_state_sub = self.create_subscription(msg.RobotStatus, '/radio/robot_status/robot_'+str(i), self.create_partial_robots, 10)
+            self.robot_intent_pubs[i] = self.create_publisher(msg.RobotIntent, '/gameplay/robot_intent/robot_'+str(i), 10)
+
         
+        self.get_logger().info("Gameplay node started")
         self.world_state = world_state
         self.partial_world_state: conv.PartialWorldState = None
         self.game_info: rc.GameInfo = None
         self.field: rc.Field = None
-        self.robot_statuses: List[conv.RobotStatus] = []
+        self.robot_statuses: List[conv.RobotStatus] = [conv.RobotStatus()]*NUM_ROBOTS*2
```

Comment:
Not *2, we only have robot statuses for our team's robots.

Reply:
This logic makes no sense to me, honestly. You can fill the opponent status objects in the world state creator by explicitly setting them to empty - that would make a whole lot more sense to me.